### DISCUSSION TITLE
A simple Squeezebox alarm icon

### DISCUSSION THREAD

**Discussion by ingolfdahl** [LMS-Community/slimserver#496] [Ideas]
I felt I needed a Squeezebox alarm icon, similar to the usual Squeezebox icon. It is useful as a symbol for a direct link to the alarm settings. Since .ICO files are not supported here, I share it as a GIF file instead. You can convert it to an .ICO file using for instance Irfanview. I do not know if this is the best way of sharing it.
![Sqeezealarm](https://user-images.githubusercontent.com/26764225/103368322-87aad800-4ac7-11eb-8dd2-47496472398d.gif)

**Comment by michaelherger** [comment]
Hi @ingolfdahl - thanks for the suggestion. Alas, I don't know where this would be used?
